### ISSUE TITLE
[Request]: Display number of installed runners per type

### ISSUE THREAD

**Issue by drws** [bottlesdevs/Bottles#2174]
### Tell us the problem or your need

In Preferences > Runners the list is hard to read since user has to first manually expand every runner type and then also manually inspect what is installed for each.

### Describe the solution you'd like

A very simple indication such as number of installed runners per type is needed so the user is helped to navigate the dull list. The number should be shown only where it is greater than zero.

**Comment by mirkobrombin** [comment]
I don't understand the request. What's the purpose of showing a counter? You still have to inspect them, no?

**Comment by drws** [comment]
Yes, the list still needs to be inspected, but if the user is hinted as to which runners are installed and doesn't need to expand every one, it is much less tiresome. Currently the user has to click on every one to check what is installed and a simple counter would help that a lot while making UI easier to read. Currently it's like reading a neverending bar menu :)

Alternatively the installed runners could be highlighted in some other way, but a simple text tag _2 installed_ gives you a lot of intormation while, again, making the UI easier to use. I see what you're doing UI-wise and I agree that most of the time less is in fact more, but opposite cases exist. I'd say this is one of them.

**Comment by mirkobrombin** [comment]
Idk I still think this feature is useless to be honest. We will evaluate

**Comment by drws** [comment]
Issue is still present in recent versions with a bit reworked GUI.

The main issue is still that user **has to click many times** simply to make a quick overview of runners. User has to expand, go through every runner list and count all installed ones. There needs to be some kind of a solution for quick overview.

@mirkobrombin Please note _some kind_ of a solution. It doesn't need to be a counter, but I couldn't think of a more practical and aesthetic solution.
